docs/Engineering/Services/data-engineering.md

```diff
@@ -0,0 +1,22 @@
+# Data Engineering
+
+Data Engineering is the part of a client's journey where they have lots of data but are not sure how to use it to make informed decisions. That's where we can help! We work with clients to understand their data needs and identify relevant sources of data. We can design and implement data pipelines and advise on different data-store technologies, such as databases, data lakes, data warehouses, ETL tools, and big data platforms.
+
+We understand the different stages involved in defining a Data Pipeline.
+
+### Data Ingestion
+Where we collect data from various sources and prepare it for further processing. Some tools used for data ingestion include Apache Kafka, AWS Kinesis, or Google Pub/Sub. For example, a retail company can collect data from point-of-sale systems, social media platforms, and customer service interactions and use Apache Kafka to ingest the data into a central data lake for analysis.
+
+### Data Processing
+Where we remove inconsistencies and errors and prepare the data for analysis. Some tools used in data processing include Apache Spark, Apache Flink, or AWS Glue. For instance, a financial institution can use Apache Spark to perform batch processing and analysis of transaction data from millions of customers for fraud detection and risk assessment.
+
+### Data Storage
+Where we store the processed data in a suitable database or data warehouse. The choice of database depends on the nature of the data, the size of the data, and the type of analysis required. Some examples of databases used for data storage include PostgreSQL, MySQL, or Amazon Redshift. For example, a logistics company can store shipping data in a MySQL database and use it for route optimisation and delivery tracking.
+
+### Data Analysis
+Where we query and analyze the stored data using various analytical tools and techniques. Some tools used in data analysis include SQL queries, Python, or R. For example, a healthcare organisation can use machine learning algorithms to analyze patient data and predict disease outcomes and treatment effectiveness.
+
+### Data Visualisation
+Where we present the results of the analysis in a visual format such as charts, graphs, or dashboards. Visualisations help make the insights obtained from the data accessible and understandable to all stakeholders. Visualisations tools such as Tableau, Power BI, or D3.js can be used to create interactive and visually appealing dashboards that can help communicate complex data insights to non-technical stakeholders. For instance, a retail company can use Tableau to create interactive dashboards to visualize sales data and monitor inventory levels across multiple locations.
+
```

Comment:
We need to also talk about the consultancy part of this service.  What are we looking for when we're on client?  What sort of outcomes are we interested in?  How do we define success?